README.md

```diff
@@ -2,79 +2,74 @@
+- [Firewall Policy](https://docs.microsoft.com/azure/firewall-manager/policy-overview#:~:text=Firewall%20Policy%20is%20an%20Azure,work%20across%20regions%20and%20subscriptions.)
+- [Private DNS Zones](https://docs.microsoft.com/azure/dns/private-dns-privatednszone#:~:text=By%20using%20private%20DNS%20zones,that%20are%20linked%20to%20it.)
+- [Container Registry](https://docs.microsoft.com/azure/container-registry/)
+- [Purview](https://docs.microsoft.com/azure/purview/)
+- [Key Vault](https://docs.microsoft.com/azure/key-vault/general)
+- [Storage Account](https://docs.microsoft.com/azure/storage/common/storage-account-overview)
+- [Synapse Private Link Hub](https://docs.microsoft.com/azure/synapse-analytics/security/synapse-private-link-hubs)
+- [PowerBI](https://docs.microsoft.com/power-bi/fundamentals/power-bi-overview)
+- [Policies](https://docs.microsoft.com/azure/governance/policy/overview)
 
 For more details regarding the services that will be deployed, please read the [Data Management](https://github.com/Azure/Enterprise-Scale-Analytics/blob/main/docs/02-datamanagement/01-overview.md) guide in the Enterprise Scale Analytics documentation.
 
 You have two options for deploying this reference architecture:
+
 1. Use the `Deploy to Azure` button for an immediate deployment
 2. Use GitHub Actions or Azure DevOps Pipelines for an automated, repeatable deployment
 
-# Prerequisites
+## Prerequisites
 
 The following prerequisites are required to make this repository work:
-* an Azure subscription
-* [User Access Administrator](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#user-access-administrator) or [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) access to the subscription to be able to create a service principal and role assignments for it.
 
-If you don’t have an Azure subscription, [create your Azure free account today](https://azure.microsoft.com/en-us/free/).
+- an Azure subscription
+- [User Access Administrator](https://docs.microsoft.com/azure/role-based-access-control/built-in-roles#user-access-administrator) or [Owner](https://docs.microsoft.com/azure/role-based-access-control/built-in-roles#owner) access to the subscription to be able to create a service principal and role assignments for it.
 
-# Option 1: Deploy to Azure - Quickstart
+If you don’t have an Azure subscription, [create your Azure free account today](https://azure.microsoft.com/free/).
+
+## Option 1: Deploy to Azure - Quickstart
 
 | Data Management Zone |
 |:---------------------|
 [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fdata-management-zone%2Fmain%2Fdocs%2Freference%2Fdeploy.dataHub.json)
 
-# Option 2: GitHub Actions or Azure DevOps Pipelines
+## Option 2: GitHub Actions or Azure DevOps Pipelines
 
 ## 1. Create repository from a template
```

Comment:
Please can we increase the level of all of the headings below Option 2

Instead of 
## 1. Create repository from a template

Can we change to
### 1. Create repository from a template